rslib/src/import_export/text/csv/metadata.rs

```diff
@@ -168,6 +161,16 @@ impl Collection {
                     metadata.guid_column = n;
                 }
             }
+            "limit dupe check to deck" => {
```

Comment:
This feels a bit verbose to me, but I'm struggling to come up with better alternatives. Maybe "duplicate scope: deck/collection"?

Reply:
No objections, but since this is hard-wired to a GUI option, I would keep the two very similar. Should I replace the GUI switch with a selector as well then?

Reply:
Thinking about it more, deck/collection is a bit misleading too. notetype/notetype+deck might better reflect what's going on, but it's wordy. I'm not sure what's best. Any other ideas? :-)

Reply:
I like `duplicate scope:notetype+deck` because it's close to the implementation and could easily support more options in the future, like `duplicate scope:deck` or `duplicate scope:all` (or `collection`).
So should we use this in the GUI, too?

Reply:
Ok, sounds good. Maybe 'notetype + deck' to avoid triggering people with OCD? :-)